index.js

```diff
@@ -5,46 +5,48 @@ var through = require('through2');
 var minimatch = require("minimatch");
 var _ = require('lodash');
 
-function instrument(options) {
-  var Instrumenter = require(resolve.sync(options.instrumenter, {basedir: __dirname}));
+function filterFiles(options, files) {
   var excludePattern = options.exclude ? [].concat(options.exclude) : [''];
```

Comment:
Since we are going to break the API anyways, should we skip `node_modules` by default here? I think that would be better.